src/ensembl/src/content/app/species/components/species-app-bar/SpeciesAppBar.tsx

```diff
@@ -14,19 +14,68 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import React, { memo, useMemo } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import isEqual from 'lodash/isEqual';
 
-import AppBar from 'src/shared/components/app-bar/AppBar';
-import { HelpPopupButton } from 'src/shared/components/help-popup';
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { AppName } from 'src/global/globalConfig';
+
+import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
+import { getEnabledCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
+
+import AppBar, {
+  HelpAndDocumentation
+} from 'src/shared/components/app-bar/AppBar';
+import { SelectedSpecies } from 'src/shared/components/selected-species';
+import SpeciesTabsWrapper from 'src/shared/components/species-tabs-wrapper/SpeciesTabsWrapper';
+
+import { RootState } from 'src/store';
+import { CommittedItem } from 'src/content/app/species-selector/types/species-search';
+
+type SpeciesAppBarProps = {
+  species: CommittedItem[];
+  activeGenomeId: string | null;
+  onSpeciesSelect: (genomeId: string) => void;
+};
+
+const SpeciesAppBar = (props: SpeciesAppBarProps) => {
+  const speciesTabs = useMemo(() => {
+    return props.species.map((species, index) => (
+      <SelectedSpecies
+        key={index}
+        species={species}
+        isActive={species.genome_id === props.activeGenomeId}
+        onClick={() => props.onSpeciesSelect(species.genome_id)}
+      />
+    ));
+  }, [props.species]);
+
+  const speciesSelectorLink = useMemo(() => {
+    return <Link to={urlFor.speciesSelector()}>Change</Link>;
+  }, []);
+
+  const wrappedSpecies = (
+    <SpeciesTabsWrapper
+      isWrappable={true}
+      speciesTabs={speciesTabs}
+      link={speciesSelectorLink}
```

Comment:
no link in the design...

Reply:
The text `Use species tabs..` seems to appear where the link gets displayed. Do you we can change this to `aside` instead of link?